android/src/main/java/com/oney/WebRTCModule/DailyWebRTCDevicesManager.java

```diff
@@ -0,0 +1,201 @@
+package com.oney.WebRTCModule;
+
+import android.content.Context;
+import android.media.AudioDeviceCallback;
+import android.media.AudioDeviceInfo;
+import android.media.AudioManager;
+import android.util.Log;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.bridge.WritableArray;
+import com.facebook.react.bridge.WritableMap;
+
+import org.webrtc.Camera1Enumerator;
+import org.webrtc.Camera2Enumerator;
+import org.webrtc.CameraEnumerator;
+
+import java.util.Arrays;
+
+public class DailyWebRTCDevicesManager {
+
+    static final String TAG = DailyWebRTCDevicesManager.class.getCanonicalName();
+    static final String ON_DEVICE_CHANGE_EVENT = "mediaDevicesOnDeviceChange";
+
+    public enum DeviceKind {
+        // Those constants are defined on the webrtc specification
+        // https://w3c.github.io/mediacapture-main/#dom-mediadevicekind
```

Comment:
except "audio" hehe

Reply:
Removed the comment since we are not following the webrtc specification anymore.